app/services/rate_limiter.rb

```diff
@@ -69,24 +69,33 @@ def maxed?
     attempts && attempts >= RateLimiter.max_attempts(rate_limit_type)
   end
 
+  def expiration_minutes
+    minutes = self.class.attempt_window_in_minutes(rate_limit_type)
+    exponential_factor = self.class.attempt_window_exponential_factor(rate_limit_type)
+    attempt_window_max = self.class.attempt_window_max_in_minutes(rate_limit_type)
```

Comment:
```suggestion
    minutes = RateLimiter.attempt_window_in_minutes(rate_limit_type)
    exponential_factor = RateLimiter.attempt_window_exponential_factor(rate_limit_type)
    attempt_window_max = RateLimiter.attempt_window_max_in_minutes(rate_limit_type)
```

I find it clearer to be explicit, and I think this still works. I don't feel strongly though.